fixtures/simple1/guardians.yaml

```diff
@@ -0,0 +1,9 @@
+data:
+  - id: 1
+    name: John Doe
+    email: john@doe.com
+    password: JohnDoe1
+  - id: 2
+    name: Nice Guy
+    email: nice@guy.com
+    password: NiceGuy1
```

Comment:
should we place a result of 
`BCrypt::Passeword.create('NiceGuy1')` and have the original password as a comment?